nifi/upload_new_nifi_version.sh

```diff
@@ -29,31 +29,41 @@ trap cleanup EXIT
 
 cd "$WORK_DIR" || exit
 
+src_file="nifi-$VERSION-source-release.zip"
 bin_file="nifi-$VERSION-bin.zip"
 download_url="https://archive.apache.org/dist/nifi/${VERSION}"
 
-echo "Downloading Nifi (this can take a while, it is intentionally downloading from a slow mirror that contains all old versions)"
+echo "Downloading Nifi source (this can take a while, it is intentionally downloading from a slow mirror that contains all old versions)"
```

Comment:
```suggestion
echo "Downloading NiFi source (this can take a while, it is intentionally downloading from a slow mirror that contains all old versions)"
```